pycbc/psd/analytical_space.py

```diff
@@ -369,19 +369,20 @@ def averaged_lisa_fplus_sq_approx(f, len_arm=2.5e9):
     -----
         Pease see Eq.(36) in <LISA-LCST-SGS-TN-001> for more details.
     """
-    from os import getcwd, path
+    from os import path
     import requests
     from scipy.interpolate import interp1d
+    import pycbc.psd
 
     if len_arm != 2.5e9:
         raise Exception("Currently only support 'len_arm=2.5e9'.")
-    cwd = getcwd()
-    if path.exists(cwd+"/AvFXp2_Raw.npy") is False:
+    module_path = pycbc.psd.__path__[0]
```

Comment:
@WuShichao A better approach is to use the astropy download_file function which can do the caching for you. 

https://docs.astropy.org/en/stable/api/astropy.utils.data.download_file.html

And should remove a number of lines in this file.